client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -108,7 +108,8 @@ class ActivityLogItem extends Component {
 
 		forEach( nextProps.pluginsToUpdate, ( plugin, key ) => {
 			if (
-				this.props.pluginsToUpdate[ key ].updateStatus.status === plugin.updateStatus.status ||
+				get( this.props.pluginsToUpdate[ key ], 'updateStatus.status', '' ) ===
```

Comment:
Since we're using `get()`, I'd personally do something like 

```js
get( this.props, [ 'pluginsToUpdate', key, 'updateStatus', 'status' ], '' )
```

That's not a blocker though.

Reply:
True. Will update.

Reply:
I used `get( this.props.pluginsToUpdate, [ key, 'updateStatus', 'status' ], '' )` because `pluginsToUpdate` is a connected prop and will be there for sure. We just want to ensure that its inner props are there.